seacrowd/sea_datasets/vsolscsum/vsolscsum.py

```diff
@@ -0,0 +1,244 @@
+}
+"""
+
+_DATASETNAME = "vsolscsum"
+
+_DESCRIPTION = """
+The Vietnamese dataset for social context summarization \
+    The dataset contains 141 open-domain articles along with \
+    3,760 sentences, 2,448 extracted standard sentences and \
+    comments as standard summaries and 6,926 comments in 12 \
+    events. This dataset was manually annotated by human. \
+    Note that the extracted standard summaries also include comments.\
+    The label of a sentence or comment was generated based on the \
+    voting among social annotators. For example, given a sentence, \
+    each annotator makes a binary decision in order to indicate \
+    that whether this sentence is a summary candidate (YES) or not \
+    (NO). If three annotators agree yes, this sentences is labeled by 3. \
+    Therefore, the label of each sentence or comment ranges from 1 to 5\
+    (1: very poor, 2: poor, 3: fair, 4: good; 5: perfect). The standard \
+    summary sentences are those which receive at least three agreements \
+    from annotators. The inter-agreement calculated by Cohen's Kappa \
+    after validation among annotators is 0.685.
+"""
+
+_HOMEPAGE = "https://github.com/nguyenlab/VSoLSCSum-Dataset"
+
+_LANGUAGES = ["vie"]
+
+_LICENSE = Licenses.CC_BY_4_0.value
+
+_LOCAL = False
+
+_URLS = {
+    _DATASETNAME: "https://raw.githubusercontent.com/nguyenlab/VSoLSCSum-Dataset/master/VSoSLCSum.xml",
+}
+
+_SUPPORTED_TASKS = [Tasks.SUMMARIZATION]
+
+_SOURCE_VERSION = "1.0.0"
+
+_SEACROWD_VERSION = "1.0.0"
+
+
+class VSolSCSumDataset(datasets.GeneratorBasedBuilder):
+    """
+    The Vietnamese dataset for social context summarization \
+    The dataset contains 141 open-domain articles along with \
+    3,760 sentences, 2,448 extracted standard sentences and \
+    comments as standard summaries and 6,926 comments in 12 \
+    events. This dataset was manually annotated by human. \
+    Note that the extracted standard summaries also include comments.\
+    The label of a sentence or comment was generated based on the \
+    voting among social annotators. For example, given a sentence, \
+    each annotator makes a binary decision in order to indicate \
+    that whether this sentence is a summary candidate (YES) or not \
+    (NO). If three annotators agree yes, this sentences is labeled by 3. \
+    Therefore, the label of each sentence or comment ranges from 1 to 5\
+    (1: very poor, 2: poor, 3: fair, 4: good; 5: perfect). The standard \
+    summary sentences are those which receive at least three agreements \
+    from annotators. The inter-agreement calculated by Cohen's Kappa \
+    after validation among annotators is 0.685.
+    """
```

Comment:
This is the same as `_DESCRIPTION`, so would you please write a more simplified description other than the one in `_DESCRIPTION`?

Reply:
Solved on https://github.com/SEACrowd/seacrowd-datahub/pull/609/commits/ccad08500fcf3ab456aea35a2c8ab7b734404d4f.